docs/integration/DataTypingAndSchemasHandling.md

```diff
@@ -143,13 +143,14 @@ properties.
 | [enum](https://json-schema.org/understanding-json-schema/reference/generic.html#enumerated-values) | String                                                |                                                                                                                                                                                          |
 | [array](https://json-schema.org/understanding-json-schema/reference/array.html)                    | [list](/docs/scenarios_authoring/Spel.md#arrayslists) |                                                                                                                                                                                          |
 
-#### [String Format](https://json-schema.org/understanding-json-schema/reference/string.html#format)
-
-| JSON Schema | Java type     | Sample                    | Comment                |
-|-------------|---------------|---------------------------|------------------------|
-| date-time   | ZonedDateTime | 2021-05-17T07:34:00+01:00 |                        |
-| date        | LocalDate     | 2021-05-17                | Timezone is not stored |
-| time        | LocalTime     | 07:34:00.12345            | Timezone is not stored |
+#### String Format
+We support the following JSON [string format](https://json-schema.org/understanding-json-schema/reference/string.html#format) keywords.
+
+| JSON Schema | Java type     | Sample                    | Comment                     |
+|-------------|---------------|---------------------------|-----------------------------|
+| date-time   | ZonedDateTime | 2021-05-17T07:34:00+01:00 | Must carry zone information |
```

Comment:
WDYT about linking the Java documentation in the column where we talk about the Java type? 
eg
https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/LocalDate.html
https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/LocalTime.html
https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/ZonedDateTime.html

Reply:
Having thought about it again - I would not add Java links you propose. The Java stuff we mention here is a kind of _under the hood_ stuff, but clearly, you do not need to know this, whether you just author scenarios or deal with complex enrichers. The link to json time and data types is there in the docs.